### PR TITLE
AT_13.10.03 | Pipeline > Pipeline Syntax > Verify Declarative Directive can be generated

### PR DESCRIPTION
**Pull request by ozhegov**
AT - https://trello.com/c/VfuUSZSk

### CODE REVIEW

**Review by lenashlyakhovets — APPROVED** [review]

File: cypress/e2e/tests/pipelineProject.cy.js [hunk 1]
```diff
@@ -187,4 +188,19 @@ describe('pipelineProject',()=>{
         .getDeclarativeOnlineDocumentationPageUrl()
         .should('contain', piplineSyntaxPageData.declarativeOnlineDocumentationURL)
     });
+
+    it('TC_13.10.03 | Verify Declarative Directive can be generated', () => {
```
Comment (IrynaKolh) [inline cypress/e2e/tests/pipelineProject.cy.js:192]: change TC to AT

File: cypress/pageObjects/DeclarativeDirectiveGeneratorPage.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        return this
+    }
+
+    clickGenerateDeclarativeDirectiveBtn() {
+        this.getGenerateDeclarativeDirectiveBtn().click()
+        return this
+    }
+}
+
+export default DeclarativeDirectiveGeneratorPage;
```
Comment (S-Lanchik) [inline cypress/pageObjects/DeclarativeDirectiveGeneratorPage.js:31]: Is this page for pipeline job only? If so, you should name it including "pipline". Fo ex, piplineDerectiveGeneratorPage.js The same for json file.

File: cypress/e2e/tests/pipelineProject.cy.js (hunk 1 above)
Comment (ozhegov) [reply cypress/e2e/tests/pipelineProject.cy.js]: @IrynaKolh done

File: cypress/pageObjects/DeclarativeDirectiveGeneratorPage.js (hunk 2 above)
Comment (ozhegov) [reply cypress/pageObjects/DeclarativeDirectiveGeneratorPage.js]: @S-Lanchik done

**Review by IrynaKolh — APPROVED** [review]

**Pull request merged**